package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": "3.6.0"
+    "version": "3.4.1"
```

Comment:
did you run `npm i`? This should be version 3.6.0.

Reply:
`npm i` didn't seem to do anything, so I had to manually change it back.